docs/en/latest/plugins/tencent-cloud-cls.md

```diff
@@ -0,0 +1,141 @@
+---
+title: tencent-cloud-cls
+keywords:
+  - APISIX
+  - API Gateway
+  - Plugin
+  - CLS
+  - Tencent Cloud
+description: This document contains information about the Apache APISIX tencent-cloud-cls Plugin。
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Description
+
+The `tencent-cloud-cls` Plugin uses [TencentCloud CLS](https://cloud.tencent.com/document/product/614)API to forward logs to your topic.
+
+## Attributes
+
+| Name              | Type    | Required | Default | Valid values  | Description                                                  |
+| ----------------- | ------- | -------- |---------| ------------- | ------------------------------------------------------------ |
+| cls_host          | string  | Yes      |         |               | CLS API host，please refer [Uploading Structured Logs](https://www.tencentcloud.com/document/api/614/16873) |
+| cls_topic         | string  | Yes      |         |               | topic of CLS                                                 |
+| secret_id         | string  | Yes      |         |               | SecretId of your API Key                                     |
+| secret_key        | string  | Yes      |         |               | SecretKey of your API KEY                                    |
+| sample_ratio      | number  | No       | 1       | [0.00001, 1]  | How often to sample the requests. Setting to `1` will sample all requests. |
+| include_req_body  | boolean | No       | false   | [false, true] | When set to `true` includes the request body in the log. If the request body is too big to be kept in the memory, it can't be logged due to Nginx's limitations. |
```

Comment:
```suggestion
| include_req_body  | boolean | No       | false   | [false, true] | When set to `true` includes the request body in the log. If the request body is too big to be kept in the memory, it can't be logged due to NGINX's limitations. |
```